data/50-default.json

```diff
@@ -85,5 +85,48 @@
     "ref-kind": "runtime",
     "name": "com.endlessm.CompanionAppService.Locale",
     "branch": "stable"
+  },
+  {
+    "action": "install",
+    "serial": 2019102100,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.Clubhouse",
+    "branch": "eos3",
+    "filters": {
+        "architecture": ["x86_64"]
+    }
+  },
+  {
+    "action": "install",
+    "serial": 2019102200,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.OperatingSystemApp",
+    "branch": "eos3",
+    "filters": {
+        "architecture": ["x86_64"]
+    }
+  },
+  {
+    "action": "install",
+    "serial": 2019102300,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.ProjectLibrary",
```

Comment:
Have you tested whether this causes `com.hack_computer.ProjectLibrary.Content` to be installed too? The reason I ask is that I installed the clubhouse from the app center, and installed these other two from the command line, and I noticed that com.hack_computer.Clippy.Extension and com.hack_computer.ProjectLibrary.Content are pulled in too. I'd expect that eos-updater-flatpak-installer would behave the same way but I'm not sure.